og_access/src/Plugin/OgFields/OgContentAccessField.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace Drupal\og_access\Plugin\OgFields;
+
+use Drupal\og\OgFieldBase;
+use Drupal\og\OgFieldsInterface;
+use Drupal\og_access\OgAccess;
+
+/**
+ * Determine the group content visibility.
+ *
+ * @OgFields(
+ *  id = OgAccess::OG_ACCESS_CONTENT_FIELD,
```

Comment:
```suggestion
 *  id = \Drupal\og_access\OgAccess\OgAccess::OG_ACCESS_CONTENT_FIELD,
```

Uncaught PHP Exception Doctrine\\Common\\Annotations\\AnnotationException: "[Semantical Error] Couldn't find constant OG_ACCESS_FIELD